AirCasting/CoreData/MeasurementStreamStorage.swift

```diff
@@ -26,7 +26,6 @@ protocol MeasurementStreamStorageContextUpdate {
 extension HiddenCoreDataMeasurementStreamStorage {
     func addMeasurementValue(_ value: Double, at location: CLLocationCoordinate2D? = nil, toStreamWithID id: MeasurementStreamLocalID) throws {
         try addMeasurement(Measurement(time: Date(), value: value, location: location), toStreamWithID: id)
-        try save()
```

Comment:
When does it gets saved now when `addMeasurementValue` is called in `SyncingMeasurementsViewModel` ?

Reply:
Each method from `HiddenCoreDataMeasurementStreamStorage` including this one must be called within `accessStorage` method:
```
    func accessStorage(_ task: @escaping(HiddenCoreDataMeasurementStreamStorage) -> Void) {
        context.perform {
            task(self.hiddenStorage)
            try? self.hiddenStorage.save()
        }
```
This way we don't have to remember to call `save()` :)

Reply:
Awesome! Does it mean we could remove save() from `MobilePeripheralSessionManager` and `MicrophoneManager` as well?

Reply:
Good catch! 🥷🏻